src/System.Collections.Concurrent/tests/ProducerConsumerCollectionTests.cs

```diff
@@ -731,25 +731,19 @@ public void ManyConcurrentAddsTakes_EnsureTrackedCountsMatchResultingCollection(
         }
 
         [Theory]
-        [InlineData(ConcurrencyTestSeconds)]
-        public void ManyConcurrentAddsTakes_CollectionRemainsConsistent(double seconds)
+        [InlineData(1000)]
```

Comment:
(Or the test needs to be reworked entirely to allow it to run for a specific amount of time without concern for growing the collection too large.)